src/foam/nanos/notification/email/EmailServiceDAO.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
```

Comment:
2019 :p , and there's a smaller header you can use
```
/**
 * @license
 * Copyright 2019 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */
```

Reply:
:|